source/authentication/overview/configure-authentication.rst

```diff
@@ -54,12 +54,6 @@ under :file:`/etc/ood/config/ood_portal.yml`. For example, to add support for
 an authentication module with ``AuthType`` of ``my-auth``, you would modify the
 :file:`/etc/ood/config/ood_portal.yml` file as such:
 
-   .. code-block:: sh
```

Comment:
@ericfranz this section seems to have been removed by Nick, and so far as I can see was not supposed to be here in the first place. I don't know the point of calling `rake` or `rake install` at this point in the documentation would be. Thoughts?

Reply:
Confirmed good delete.